INFO 2025-01-28 18:33:27,655 autoreload Watching for file changes with StatReloader
INFO 2025-01-28 18:33:43,888 autoreload C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\tamarcado\settings\base.py changed, reloading.
INFO 2025-01-28 18:38:09,241 autoreload Watching for file changes with StatReloader
INFO 2025-01-28 18:38:11,990 brasil_api Fazendo requisi豫o para BrasilAPI para a data: 2026-01-01
INFO 2025-01-30 14:37:07,749 autoreload Watching for file changes with StatReloader
INFO 2025-02-06 17:29:28,173 autoreload Watching for file changes with StatReloader
WARNING 2025-02-06 17:30:45,045 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 17:32:59,401 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 17:35:10,923 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 17:35:45,474 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 17:35:53,938 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 17:35:57,890 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 17:56:44,675 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 17:56:46,534 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 17:57:01,781 log Forbidden: /api/agendamentos/
INFO 2025-02-06 17:57:49,537 autoreload Watching for file changes with StatReloader
WARNING 2025-02-06 18:09:19,732 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 18:11:29,880 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 18:14:06,255 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 18:14:15,424 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 18:16:20,102 log Forbidden: /api/agendamentos/
WARNING 2025-02-06 18:18:10,628 log Forbidden: /api/agendamentos/
INFO 2025-02-07 19:32:21,995 autoreload Watching for file changes with StatReloader
INFO 2025-02-07 19:32:25,486 brasil_api Fazendo requisi豫o para BrasilAPI para a data: 2026-01-20
WARNING 2025-02-07 19:32:25,959 log Bad Request: /api/agendamentos/
INFO 2025-02-07 19:32:38,768 brasil_api Fazendo requisi豫o para BrasilAPI para a data: 2026-12-25
INFO 2025-02-07 19:32:46,589 brasil_api Fazendo requisi豫o para BrasilAPI para a data: 2026-02-25
WARNING 2025-02-07 19:33:02,308 log Forbidden: /api/agendamentos/
INFO 2025-02-07 19:34:07,116 autoreload C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\agenda\serializers.py changed, reloading.
INFO 2025-02-07 19:34:07,822 autoreload Watching for file changes with StatReloader
INFO 2025-02-07 19:42:19,706 autoreload C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\agenda\views.py changed, reloading.
INFO 2025-02-07 19:42:20,223 autoreload Watching for file changes with StatReloader
INFO 2025-02-07 20:25:05,734 autoreload C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\agenda\views.py changed, reloading.
INFO 2025-02-07 20:25:07,369 autoreload Watching for file changes with StatReloader
INFO 2025-02-19 19:52:21,660 autoreload Watching for file changes with StatReloader
ERROR 2025-02-19 19:52:52,166 log Internal Server Error: /api/prestadores/
Traceback (most recent call last):
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 511, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 423, in finalize_response
    assert isinstance(response, HttpResponseBase), (
AssertionError: Expected a `Response`, `HttpResponse` or `StreamingHttpResponse` to be returned from the view, but received a `<class 'NoneType'>`
INFO 2025-02-19 20:04:34,948 autoreload C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\agenda\views.py changed, reloading.
INFO 2025-02-19 20:04:35,905 autoreload Watching for file changes with StatReloader
INFO 2025-02-19 20:05:12,438 autoreload C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\agenda\views.py changed, reloading.
INFO 2025-02-19 20:05:13,544 autoreload Watching for file changes with StatReloader
ERROR 2025-02-19 20:05:35,325 log Internal Server Error: /api/prestadores/
Traceback (most recent call last):
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\agenda\views.py", line 92, in relatorio_prestadores
    response = HttpResponse(
               ^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\http\response.py", line 374, in __init__
    super().__init__(*args, **kwargs)
TypeError: HttpResponseBase.__init__() got an unexpected keyword argument 'headears'
ERROR 2025-02-19 20:06:22,255 log Internal Server Error: /api/prestadores/
Traceback (most recent call last):
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\agenda\views.py", line 92, in relatorio_prestadores
    response = HttpResponse(
               ^^^^^^^^^^^^^
  File "C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\venv\Lib\site-packages\django\http\response.py", line 374, in __init__
    super().__init__(*args, **kwargs)
TypeError: HttpResponseBase.__init__() got an unexpected keyword argument 'headears'
INFO 2025-02-19 20:07:18,770 autoreload C:\Users\Thiago\OneDrive\햞ea de Trabalho\Py-Django\dfr-api\agenda\views.py changed, reloading.
INFO 2025-02-19 20:07:19,402 autoreload Watching for file changes with StatReloader
